modules/mono/managed_callable.h

```diff
@@ -66,7 +67,7 @@ class ManagedCallable : public CallableCustom {
 
 	void release_delegate_handle();
 
-	ManagedCallable(GCHandleIntPtr p_delegate_handle);
+	ManagedCallable(GCHandleIntPtr p_delegate_handle, ObjectID p_object);
```

Comment:
For clarity:

```suggestion
	ManagedCallable(GCHandleIntPtr p_delegate_handle, ObjectID p_object_id);
```

Same for all other occurrences in this PR.